core/src/test/java/org/testcontainers/containers/VncRecordingContainerTest.java

```diff
@@ -0,0 +1,33 @@
+package org.testcontainers.containers;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class VncRecordingContainerTest {
+
+    private GenericContainer target;
+
+    @Before
+    public void initialize() {
+        target = new GenericContainer("selenium/standalone-chrome-debug").
```

Comment:
1) `.` on a next line
2) you can add `target` to the `try-with-resources` in `verifyStartup`, no need to have it in `@Before`/`@After`